dimod/discrete/cydiscrete_quadratic_model.pyx

```diff
@@ -261,6 +261,36 @@ cdef class cyDiscreteQuadraticModel:
 
         return biases
 
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    cpdef void add_constraint_as_quadratic(self, vector[VarIndex] variables, vector[CaseIndex] cases,
+                                           vector[Bias] biases, Bias lagrange_multiplier, Bias constant):
+        if len(variables) != len(cases) or len(variables) != len(biases):
+            raise ValueError("There should be equal number of variables, "
```

Comment:
I know it's pedantic, but I think this exception should just be "There should be an equal number of variables, cases and biases". The 3-tuple part is not applicable for those who are accessing this method directly - it only applies to the DQM method, _not_ the cyDQM one. Also, since the user is giving the input as three-tuples in the other method, this error should never be seen when calling from the DQM method.